examples/serve/misc/cancel/README.md

```diff
@@ -33,7 +33,7 @@ Client disconnected, stopping computation.
 You can also run
 
 ```bash
-curl -L http://<endpoint>/
+curl http://<endpoint>/
 ```
 
 and manually Ctrl + C to cancel the request and see logs.
```

Comment:
Can we use an OpenAI API client to stream the output and check if abortion/cancellation work?

Reply:
The smoke test `test_skyserve_cancel` passed. Do you think that is enough?

Reply:
I mean we should test with the integration with OpenAI API for both [client](https://platform.openai.com/docs/guides/text-generation/chat-completions-api) and server (`vllm.entrypoints.openai.api_server`) side to make sure our solution works for the most commonly used library.

Reply:
Just tested with the [example of llama2](https://github.com/skypilot-org/skypilot/tree/master/examples/serve/llama2) (both service YAML using vLLM and openai client) w/ and w/o streaming and it works well 🫡

Reply:
Does the abortion work for it as well?